ocrd_page_to_alto/utils.py

```diff
@@ -49,3 +49,18 @@ def get_nth_textequiv(reg_page, textequiv_index, textequiv_fallback_strategy):
     else:
         return textequivs[textequiv_index].Unicode
 
+def contains(el, bbox):
```

Comment:
If we move forward with [ALTO support in core](https://github.com/OCR-D/core/pull/692) this might be a useful addition to `ocrds_utils`.